lua/astrocommunity/docker/lazydocker/init.lua

```diff
@@ -1,22 +1,23 @@
+---@type LazySpec
 return {
   {
     "mgierada/lazydocker.nvim",
     dependencies = {
       "akinsho/toggleterm.nvim",
       {
         "AstroNvim/astrocore",
-        opts = {
-          mappings = {
-            n = {
-              ["<Leader>t" .. "d"] = {
-                function() require("lazydocker").open() end,
-                desc = "ToggleTerm LazyDocker",
-              },
-            },
-          },
-        },
+        opts = function(_, opts)
+          local maps = opts.mappings
+
+          -- INFO: Default prefix of toggleterm-related actions
+          local prefix = "<Leader>t"
+
+          maps.n[prefix .. "d"] = {
+            function() require("lazydocker").open() end,
+            desc = "ToggleTerm LazyDocker",
+          }
+        end,
       },
     },
-    opts = {},
```

Comment:
this would run, the `setup()` of `lazydocker.nvim` 
https://github.com/crnvl96/lazydocker.nvim?tab=readme-ov-file#requirements

Reply:
Yeah this is good

Reply:
@luxus yeah, I also checked what happens inside `lazydocker.nvim` if opts are `nil`, and it [will replaced by `{}`](https://github.com/mgierada/lazydocker.nvim/blob/38c0fbc3cb77e38fd2cdc70b709cf841a38782f5/lua/lazydocker/config.lua#L20-L22).

Reply:
If you don't have it here the setup function is never called. This is necessary to have :) or you can change it to `config = true` which will pass in `nil` to the setup function. But it's better to just leave it untouched rather than breaking it by removing it